compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +64,25 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getPeelAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, "enable_loop_peeling");
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface *funcOp) {
```

Comment:
This does not need to be a `FunctionOpInterface *`. Thats not the norm. Usually just pass these by value.